File: Programmers/Lv.0/k의 개수/120887.js
function solution(i, j, k) {
    let answer = 0;
    
    for (let a=i; a<=j; a++) {
        const kstr = k.toString();
        const astr = a.toString().split('');
        astr.forEach(v => {
            if (kstr === v) answer++
        });
    }
    
    return answer;
}

File: Programmers/Lv.0/k의 개수/120887.java
class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;
        
        for (int a=i; a<=j; a++) {
            String kstr = String.valueOf(k);
            String[] astr = String.valueOf(a).split("");
            for (String s : astr) {
                if (kstr.equals(s)) answer++;
            }
        }
        
        return answer;
    }
}
